Public/Src/FrontEnd/TypeScript.Net/TypeScript.Net/Types/Nodes/NodeInterfaces.cs

```diff
@@ -879,6 +879,9 @@ public interface ISwitchExpression : IExpression
     [NodeInfo(SyntaxKinds = new[] { SyntaxKind.SwitchExpressionClause }, NodeType = NodeType.Leaf)]
     public interface ISwitchExpressionClause : IExpression
     {
+        /// <nodoc/>
+        bool IsDefaultFallthrough { get; set; }

```

Comment:
>IsDefaultFallthrough [](start = 13, length = 20)

If anything in this PR deserves a bit more documentation, I think it would be this property :)